promotion/avro.go

```diff
@@ -51,3 +51,22 @@ type AdminAttestationEvent struct {
 	Justification string `json:"justification"`
 	CreatedAt     string `json:"created_at"`
 }
+
+const grantRewardsEventSchema = `{
+	"type": "record",
+	"name": "grantRewards",
+	"doc": "This message is from when a client claimed a swap reward",
+	"fields": [
+		{ "name": "promotion_id", "type": "string" },
+		{ "name": "address_id", "type": "string" },
```

Comment:
What was the thinking on the `address_id` name?  Wondering if `blockchain_address` be more more descriptive

Reply:
I'm fine with either. Should we discuss this in the doc or a channel or just do it?